### DISCUSSION TITLE
markdown中考虑加入代码折叠的功能吗?

### DISCUSSION THREAD

**Discussion by huhx** [vuepress-theme-hope/vuepress-theme-hope#4090] [Q&A] [answered]
Markdown Enhance中的code怎样添加折叠功能？

**Comment by Mister-Hope** [comment, marked as answer]
你考虑仔细阅读一下文档么？毕竟这是个开源项目，so you don't pay me

https://theme-hope.vuejs.press/zh/guide/markdown/stylize/hint.html
